test/test.yaml

```diff
@@ -17,6 +14,17 @@ printer:
   default_wait_after_exposure: 1.5
   pause_lift: 100
 
+# This section holds fields pertaining to the display used by the printer
+display:
+  #frame_buffer: /tmp/emulatedFramebuffer # Use with fbEmulator.py
+  frame_buffer: /dev/null # Dump frames to null for testing
+  bit_depth:
```

Comment:
CC: Maybe we can assume 565 as the default? Afaik most of these screen drivers use RGB565.

Reply:
See above